frontend/src/pages/CustomerProfile.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import '../App.css';
+import logo from '../TIPSlogo.png';
+import { BrowserRouter as Router, Route, Switch, Link } from 'react-router-dom';
+import { useHistory } from 'react-router';
+// having trouble displaying the contents of this page
+const CustomerProfile = () =>{
+    const history = useHistory();
+    return (
+        <Router>
+          <Switch>
+                <Route path="/customer-profile">
+                <div className="App">
+                    <div className="App-background">
+                        <p> You are viewing a sample customer profile page </p>
+                        <div>
+                            <div className="back-button" onClick={() => {history.goBack();}}> back </div>
+                        </div> 
+                        <div>
+                            <Link className="next-button" to="/signup2"> next </Link> 
```

Comment:
Since you don't actually have a route for this button, I'm assuming it's here by mistake?